tests/handlers/test_register.py

```diff
@@ -187,18 +187,30 @@ def test_auto_create_auto_join_rooms_when_support_user_exists(self):
 
     @defer.inlineCallbacks
     def test_auto_create_auto_join_where_no_consent(self):
-        """XXX what is this trying to test? I *think* it is trying to test
-        that we are not auto-joined to the server notices room if
-        block_events_without_consent_error is set, but (a) that doesn't seem to be
-        a sensible thing to test for, and (b) it doesn't work anyway because you can't
-        change the config after the EventCreationHandler has been instantiated.
+        """Test to ensure that the first user is not auto-joined to a room if
+        they have not given general consent.
         """
         self.hs.config.user_consent_at_registration = True
```

Comment:
does this and the next line actually do anything? I'd expect them to be redundant now.

Reply:
Nope